.github/workflows/v10-sync-generated-files.yml

```diff
@@ -0,0 +1,40 @@
+name: Sync generated files
```

Comment:
I think you can remove this one honestly if you want to, I think it's disabled ever since a change to the permissions for the token